### ISSUE TITLE
Responses to 1C opening

### ISSUE THREAD

**Issue by Mordermi** [hildjj/bidding#1]
Noted that 1D = 0-7 HCP, while 2H/S and 3X = 4-7 HCP and 6 or 7+ cards in suit named. Assume this means we do not care about 4-7 HCP and 5H/S, correct? Should we add the additional info about lack of suit length to 1D alert explanation?

**Comment by hildjj** [comment]
I agree that 4-7hcp and 5H/S is a 1D response.   I don't think we need to add that to the explanation, there are lots of things the 1D response isn't, and they can ask about those.  We don't say "all hands that are 0-7", we say "he has 0-7 or an unusual hand, says nothing about diamonds"

**Comment by hildjj** [comment]
And now that 1d is always 0-7 in transfer-land, I think this is even more moot.  Closing.
